.github/workflows/cla.yml

```
@@ -0,0 +1,172 @@
+      )
+    runs-on: ubuntu-latest
+    permissions:
+      actions: write
+      contents: write
+      issues: write
+      pull-requests: write
+    # Signature and lock writes share repository state. Queue every pending
+    # event so a newer comment cannot silently replace an earlier request.
+    concurrency:
+      group: cla-signatures
+      queue: max
```

actionlint (1.7.12)

[error] unexpected key "queue" for "concurrency" section. expected one of "cancel-in-progress", "group"

(syntax-check)